crates/adkg/src/rbc/r4/handlers.rs

```diff
@@ -186,6 +219,10 @@ pub(super) async fn rbc_receive_ready<T>(
         // Try to decode
         if let Some(m) = rs_decode_stripes(&stripes, &missing_indices, st.n, st.t + 1) {
             if Sha3_256::digest(&m)[..] == h[..] {
```

Comment:
I guess you're using Sha3_256 consistently throughout the package? If so my previous comment might be moot, but we want to make sure we use the same hash for the same purpose everywhere, and aren't sometimes taking the one from the scheme and sometimes hardcoding it.